lib/clients/authclient.py

```diff
@@ -0,0 +1,94 @@
+'''
```

Comment:
Is this different from lib/biokbase/ServiceWizard/authclient.py ?

Reply:
Shouldn't be, unless this PR is so old that it predates the auth2 cutover.